sklearn/linear_model/tests/test_logistic.py

```diff
@@ -2262,18 +2262,18 @@ def test_lr_cv_scores_differ_when_sample_weight_is_requested():
     sample_weight[: len(y) // 2] = 2
     kwargs = {"sample_weight": sample_weight}
 
-    scorer1 = get_scorer("accuracy")
+    scorer1 = get_scorer("accuracy").set_score_request(sample_weight=False)
```

Comment:
now that scorers properly request their routing, this is required since we're passing sample weight bellow.